types/audit.d.ts

```diff
@@ -64,23 +70,31 @@ declare global {
     export interface Product {
       rawValue: boolean | number | null;
       displayValue?: string;
+      /** An explanation of audit-related issues encountered on the test page. */
       explanation?: string;
+      /** Error message from any exception thrown while running this audit. */
       errorMessage?: string;
       warnings?: string[];
+      /** The scored value determined by the audit, in the range `0-1`, or null if `scoreDisplayMode` indicates not scored. */
       score?: number;
+      /** Deprecated and does not make its way into the Lighthouse report. */
       extendedInfo?: {[p: string]: any};
       /** Overrides scoreDisplayMode with notApplicable if set to true */
       notApplicable?: boolean;
+      /** Extra information provided by some types of audits. */
```

Comment:
seems about as descriptive as `details` ;)

do we want to talk about how this is structured data for the renderer?

Reply:
> seems about as descriptive as `details` ;)

Fair :)

> do we want to talk about how this is structured data for the renderer?

 I took a two second stab at it :)